src/taskcreator.js

```diff
@@ -46,6 +48,18 @@ class TaskCreator {
     return task;
   }
 
+  async appendLastFire({hookGroupId, hookId}, {firedBy}, lastFire) {
```

Comment:
It might be good to simplify this calling convention to just `{hookGropuId, hookId, taskId, taskCreateTime, firedBy, result, error}` and pull those properties from the various required places in the `fire` method.